packages/browser-sdk/src/hooksManager.ts

```diff
@@ -0,0 +1,105 @@
+import { CheckEvent, RawFeatures } from "./feature/features";
+import { CompanyContext, UserContext } from "./context";
+
+/**
+ * Hook for check event.
+ */
+export type CheckHook = {
+  type: "check-is-enabled" | "check-config";
+  callback: (checkEvent: CheckEvent) => void;
+};
+
+/**
+ * Hook for company update event.
+ */
+export type CompanyHook = {
+  type: "company";
+  callback: (company: CompanyContext) => void;
+};
+
+/**
+ * Hook for user update event.
+ */
+export type UserHook = {
+  type: "user";
+  callback: (user: UserContext) => void;
+};
+
+/**
+ * Hook for features updated event.
+ */
+export type FeaturesUpdatedHook = {
+  type: "features-updated";
+  callback: (features: RawFeatures) => void;
+};
+
+type trackEvent = {
```

Comment:
Nit: types should be PascalCased even if not exported

Reply:
fixed!